yt/yt/server/controller_agent/controllers/experiment_job_manager.cpp

```diff
@@ -218,14 +218,15 @@ void TExperimentJobManager::OnJobScheduled(const TJobletPtr& joblet)
     }
 }
 
-void TExperimentJobManager::OnJobCompleted(const TJobletPtr& joblet)
+bool TExperimentJobManager::OnJobCompleted(const TJobletPtr& joblet)
```

Comment:
What is semantic of this boolean? 

And it seems that we always return true (in this and all other implementations)

Reply:
MultiJobManager returns `false` when it is not finished yet.